### ISSUE TITLE
[Enhancement] Add printing of stacktrace to Try/Except in search.py line 150

### ISSUE THREAD

**Issue by devinjparsons** [AHAAAAAAA/PokemonGo-Map#1383]
## Expected Behavior
Print out the stack trace when this Try/Except catches an exception, so that the user can more easily report problems or diagnose them.

## Current Behavior
Just prints out there was an error, restarting.  Which isn't helpful at all because of the amount of places an exception could be occuring.

## Possible Solution
Either log another line with the stack trace, or change the log statement to be an error and include exc_info=True in the statement.

**Comment by panix187** [comment]
I agree.  I tried running this version but kept getting a crash with no explanation.  I added in print_exc() to figure out it was an SSL error.

Side note:  My python isn't super duper leet hax0r level.  Any way to turn off SSL cert verification?  I was able to ignore it in v1 since it used urllib3 but I'm not sure what is logging into Google now so I don't know what to search for. :\
